commit/merkleroot/observation.go

```diff
@@ -105,7 +105,7 @@ func (p *Processor) prepareRMNController(ctx context.Context, lggr logger.Logger
 		return nil
 	}
 
-	lggr.Infow("Initializing RMN controller", "rmnRemoteCfg", prevOutcome.RMNRemoteCfg)
+	lggr.Debugw("Initializing RMN controller", "rmnRemoteCfg", prevOutcome.RMNRemoteCfg)
```

Comment:
Can we move this to the RMN controller and only when an init happens, not when the controller is already inited?